packages/lab/src/stepper-input/internal/useSpinner.ts

```diff
@@ -22,12 +24,12 @@ function useSpinner(activationFn: () => void, isAtLimit: boolean) {
     };
   }, []);
 
-  const activate = () => {
-    activationFn();
+  const activate = (event: SyntheticEvent) => {
+    activationFn(event);
     setButtonDown(true);
   };
 
-  useInterval(activationFn, buttonDown ? INTERVAL_DELAY : null);
+  useInterval(() => activationFn(), buttonDown ? INTERVAL_DELAY : null);
```

Comment:
Have you tried memorizing `event`?

If it doesn't work, need to update `onChange` event to be optional